### PR TITLE
[INVE-782] Documentation - update for map layer security

### PR DESCRIPTION
**Pull request by jaroslawjanas**
[INVE-782](https://sirensolutions.atlassian.net/browse/INVE-782)

### CODE REVIEW

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 1]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+
+image::ac_map_layers_read_permissions.png[image]
+
+NOTE: This will match all map indices. You can also add each index individualy.
+
+==== Document Level Security (DLS)
+
+Document level security can be done on the map indices, this will only allow documents matching the DLS query to be returned.
+
+WARNING: To maintain good performance you should run the dls map queries on the map indices, separetly of other indices, link:#full_dls_example[example]. More on that link:https://docs.search-guard.com/latest/document-level-security#performance-considerations[ here].
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:554]:
```suggestion
[CAUTION]
====
To maintain system performance, run DLS map queries ONLY on the map indices. For more information, see the link:https://docs.search-guard.com/latest/document-level-security#performance-considerations[Search Guard performance considerations].
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 2]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+
+*By spatial path*
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:556]:
```suggestion
*Configuring security by spatial path*
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 3]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+
+This dls query will only allow documents with _"World Lakes"_ in their `spatial_path`.
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:558]:
```suggestion
The following DLS query retrieves only the documents that contain _"World Lakes"_ in their `spatial_path` parameter:
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 4]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+    fls: []
+    masked_fields: []
+    allowed_actions:
+      - READ
+      - VIEW_INDEX_METADATA
+....
+
+image::dls_spatialpath_add_layers.png[image]
+
+*By geo-shape*
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:585]:
```suggestion
*Configuring security by geo-shape*
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 5]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+
+This dls query will only allow documents within the specified co-ordinates.
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:587]:
```suggestion
The following DLS query retrieves only the documents that are within the specified coordinates:
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 6]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+  allowed_actions:
+    - READ
+    - VIEW_INDEX_METADATA
+....
+
+image::dls_geoshape_coordinates.png[image]
+
+image::dls_geoshape_map_ie.png[image]
+
+*By properties fields*
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:608]:
```suggestion
*Configuring security by properties fields*
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 7]
```diff
@@ -535,3 +535,88 @@ Layers can be toggled on and off by clicking the checkboxes
...
+
+This dls query will only allow documents that match a specific property.
```
Comment (louise-glavey) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:610]:
```suggestion
The following DLS query retrieves only the documents that match a specific property:
```

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc (hunk 1 above)
Comment (jaroslawjanas) [reply docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:557]: This gets rid of `link:#full_dls_example[example]` ?

Comment (louise-glavey) [reply docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:557]: Yes, I don't think the link is essential @jaroslawjanas

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 8]
```diff
@@ -515,7 +515,7 @@ Clicking the Add button will add the selected layers to the map, while the Add a
 If a Polygon is clicked when loaded on the map, Geo Filter(s) will be created
 
 === Layer Control
-The Layer Control is located in the top right of any Enhanced Coordinate Map. An example is in the image below showing a case where some Stored Layers are loaded (See the <<Loading stored layers from Elasticsearch>> for more details). 
+The Layer Control is located in the top right of any Enhanced Coordinate Map. An example is in the image below showing a case where some Stored Layers are loaded (See the <<Loading Stored Layers from Elasticsearch Onto Enhanced Coordinate Map, Loading stored layers from Elasticsearch>> for more details).
```
[the remark this answers is not in the document]
Comment (jaroslawjanas) [reply docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:518]: Makes sense.

**Comment by jaroslawjanas** [comment]
@louise-glavey 
All done.

**Pull request merged**